hammers/scripts/floatingip_reaper.py

```diff
@@ -24,13 +24,24 @@
 from hammers.slack import Slackbot
 from hammers.util import base_parser
 
+SAGE_IP_WHITELIST = [
```

Comment:
Maybe note which site these IPs are on. I assume it is UC? Do we need any at TACC? KVM?

Reply:
Updated to SAGE_UC_IP_WHITELIST